invenio_circulation/transitions/base.py

```diff
@@ -194,6 +194,9 @@ def execute(self, loan, transition_kwargs=None, **kwargs):
 
     def after(self, loan, initial_loan, transition_kwargs=None, **kwargs):
         """Commit record and index."""
+        if not transition_kwargs:
```

Comment:
I am not sure if this can do exactly what you want, I maybe wrong:
`def after(self, loan, initial_loan, transition_kwargs={}, **kwargs):`

Reply:
I prefer to avoid declaring mutable variables as default arguments

Reply:
what about:
```suggestion
        transition_kwargs = transition_kwargs or {}
```